easybuild/easyconfigs/c/Cython/Cython-3.0.7-GCCcore-12.3.0.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonPackage'
+
+name = 'Cython'
+version = '3.0.7'
+
+homepage = 'https://cython.org/'
+description = """
+Cython is an optimising static compiler for both the Python programming
+language and the extended Cython programming language (based on Pyrex).
+"""
+docurls = [
+    'https://cython.org/#documentation',
+    'https://github.com/cython/cython',
+]
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+builddependencies = [
+    ('binutils', '2.40'),
+]
+
+dependencies = [
+    ('Python', '3.11.3'),
+]
+
+sources = [SOURCE_TAR_GZ]
+checksums = ['fb299acf3a578573c190c858d49e0cf9d75f4bc49c3f24c5a63804997ef09213']
```

Comment:
Move to between toolchain and builddeps lines.

Reply:
@akesandgren fixed in dd9ffb19d3